pkgs/servers/home-assistant/custom-lovelace-modules/opensprinkler-card/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchurl,
+}:
+
+let
+  # Upstream does not version control a lockfile:
+  # <https://github.com/rianadon/opensprinkler-card/issues/25>
+  # This lockfile comes from <https://github.com/rianadon/opensprinkler-card/pull/26>
+  packageLockJson = fetchurl {
+    url = "https://raw.githubusercontent.com/rianadon/opensprinkler-card/be4424d92b6edbda2418ecb7462dc4c3fb8b5d13/package-lock.json";
+    hash = "sha256-sEH9D5g9WQ6+qDGYGCAXteeylBJLb4XVgTMPCUAY5qo=";
+  };
```

Comment:
This is me trying to avoid vendoring a `package-lock.json` in nixpkgs.

Let me know if it would be better for me to just copy the file here.

Reply:
This is dealt with.